fuzzers/aflsmart/builder.Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG parent_image=gcr.io/fuzzbench/base-builder
+FROM $parent_image
+
+# install AFLSmart dependencies
+RUN dpkg --add-architecture i386 && \
+    apt-get update -y && \
+    apt-get install apt-utils -y && \
```

Comment:
nit: it's probably cleaner and faster to install each package in one invocation of `apt-get install -y` rather than invoking it so many times.

Reply:
I fixed it in the new commit